free-2.md

```diff
@@ -10,34 +10,46 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The need is for professors and students to be able to communicate problem sets and solutions more easily and through technology. I would be helping both teachers and students. Right now, for a student that does their problem sets in latex, someone needs to pick up the problem set in hard copy or print it out, then write it up in latex, and then print out the typesetted version to hadn it in. With my idea this same student would be able to receive the problem set as latex, do it in latex, and then submit it in latex (potentially with a wrapper around latex to make it a bit easier?)
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL is appropriate because it would work with latex and could be written by both teachers and students to communicate uniformly. It addresses the need by removing the need to print!
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+I hate printing my math homework every day. I came up with it when I was doing no-computer in conjunction with the ideas from past years.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Students and professors of quantitatives.
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+A user would interact with this language by either:
+a) providing problems, due dates, hints, and resources through the language in order to dictate a problem set to students
+or
+b) providing solutions in wrapped-up latex.
+Programming looks different depending on which interface is being used but im imagining it looking pretty much like english but with keywords and stuff.
+This is the right way to interact with the problem domain because problem sets change every time but there is generally a fixed way of setting them up. This means that a language is probably a good solution because we can dictate the syntax without limiting the content.
 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When a program runs either students would be able to access a problem set or a grader would be able to have access to students solutions. A program interacts with the user through a log similar to latex I think and error messages are communicated through that. Errors might occur if keywords are omitted or something, and also students could be given feedback on wrong solutions through the language as well.
```

Comment:
Hmm I know we call a lot of things languages but I don't really see it here. This seems more like a program where people upload something and get comments on it. I don't see how a language naturally fits into the domain.